### PR TITLE
Fix mistakes in quantsim_code_example.py

### PR DESCRIPTION
**Pull request by shiannn**
Modify the variable `sim` to `quant_sim`
Fix the definition of `pass_calibration_data` to acquire arguments for forwards

### CODE REVIEW

File: Docs/torch_code_examples/quantsim_code_example.py [hunk 1]
```diff
@@ -46,12 +46,13 @@
 # End of PyTorch imports
 
 
-def pass_calibration_data(sim_model):
+def pass_calibration_data(sim_model, args=None):
```
Comment (shiannn) [inline Docs/torch_code_examples/quantsim_code_example.py]: Need the extra parameter to receive arguments passed for forwards

File: Docs/torch_code_examples/quantsim_code_example.py [hunk 2]
```diff
@@ -112,11 +113,11 @@ def quantize_and_finetune_example():
...
     # The following line of code illustrates that the model is getting finetuned.
     # Replace the following finetune() unction with your pipeline's finetune() function.
-    ImageNetDataPipeline.finetune(sim.model, epochs=1, learning_rate=5e-7, learning_rate_schedule=[5, 10],
+    ImageNetDataPipeline.finetune(quant_sim.model, epochs=1, learning_rate=5e-7, learning_rate_schedule=[5, 10],
                                   use_cuda=use_cuda)
 
     # Determine simulated accuracy
-    accuracy = ImageNetDataPipeline.evaluate(sim.model, use_cuda)
+    accuracy = ImageNetDataPipeline.evaluate(quant_sim.model, use_cuda)
     print(accuracy)
```
Comment (shiannn) [inline Docs/torch_code_examples/quantsim_code_example.py:121]: `sim` should be a typo.

**Comment by shiannn** [comment]
Fix #2063

File: Docs/torch_code_examples/quantsim_code_example.py (hunk 1 above)
Comment (quic-mangal) [inline Docs/torch_code_examples/quantsim_code_example.py]: Can we rename args to forward_pass_args

**Comment by quic-mangal** [comment]
@shiannn Thanks for catching and fixing these.

File: Docs/torch_code_examples/quantsim_code_example.py (hunk 1 above)
Comment (shiannn) [reply Docs/torch_code_examples/quantsim_code_example.py]: Sure. Just renamed it :)

**Review by quic-mangal — APPROVED** [review]

**Review by quic-akhobare — APPROVED** [review]

**Comment by quic-akhobare** [comment]
Thanks @shiannn. Appreciate your help.

**Pull request merged**